app/views/dogs/manager/_manager.html.erb

```diff
@@ -27,7 +40,7 @@
   </div>
 </div>
 
-<div id="manager_dogs">
+<div id="manager_dogs" class="col-lg-12">
```

Comment:
Minor update so that the list is aligned with the other rows (sorry my OCD kicked in).